python/lsst/ip/isr/isrFunctions.py

```diff
@@ -219,8 +219,56 @@ def growMasks(mask, radius=0, maskNameList=['BAD'], maskValue="BAD"):
         fpSet.setMask(mask, maskValue)
 
 
-def maskITLEdgeBleed(ccdExposure, badAmpDict, itlEdgeBleedSatMinArea=10000,
-                     itlEdgeBleedSatMaxArea=100000,
+def getLargeSatMidPoint(fpCore):
```

Comment:
This should be an underscore method because it's not in the public interface.  (It's also not in `__all__` which is good, but that would make it extra clear)

Reply:
What do you mean it is not in the public interface? It is called in isrTaskLSST

Reply:
Oh, I didn't realize that.  Then it should be in `__all__`.